tests/Integration/Monitoring/Test_MonitoringReporter.py

```diff
@@ -881,3 +918,18 @@ def test_addPilotSubmissionRecords():
     result = pilotMonitoringReporter.commit()
     assert result["OK"]
     assert result["Value"] == len(pilotMonitoringData)
+
+
+def test_addDataOperationRecords():
+    for record in dataOpMonitoringData:
+        dataOpMonitoringReporter.addRecord(record)
+    result = dataOpMonitoringReporter.commit()
+    assert result["OK"]
+    assert result["Value"] == len(dataOpMonitoringData)
+
+
+@pytest.mark.parametrize(("commitFlag, delayedCommit"), [(False, False), (True, False), (True, True), (False, True)])
+def test_DataOperationSender(commitFlag, delayedCommit):
+    for record in dataOpMonitoringData:
+        result = DataOperationSender.sendData(record, commitFlag, delayedCommit)
+    assert result["OK"], result["Message"]
```

Comment:
you are still missing a tab:
```suggestion
        assert result["OK"], result["Message"]
```